content/articles/how-to-load-and-query-a-csv-file-created-with-python-in-ibm-db2-database/index.md

```diff
@@ -0,0 +1,176 @@
+
+### How to load and query a csv file created with python in IBM db2 database.
+
+#### In this article you will learn how to 
+- Create a CSV file in python
+- Load data into IBM Db2 database
+- Run SQL queries with your CSV dataset in IBM db2 database
+
+### Introduction
+A Comma Separated Values (CSV) file are semi-structure data files which can be transformed to structured data for an ordinary user to understand the meaning of the the data in it. This is where the IBM Db2 database comes in to play, it can collect semi-structure data and transform it into a structured data and also gives you the convenient to query the data to get the required result you need.
+
+### Prerequisite 
+- python 3.10 or newest version installed on your machine.
+- working knowledge on SQL and Python.
+- IBM Db2 database account (In this tutorial we will use the free version).
+
+### Creating a CSV file with python.
```

Comment:
Hey the topic I was approved to write was "How to load and query a csv file created with **python** in IBM db2 database."

Since you have said that python part isn't needed, how can I change the URL  from **"How to load and query a csv file created with **python** in IBM db2 database."**        to    **"How to load and query a csv file in IBM db2 database."**      in this PR without creating another PR ?
Is that possible?